apps/web/src/ee/billing/components/PlanHeader.tsx

```diff
@@ -34,7 +37,7 @@ export const PlanHeader = () => {
   const [apiServiceLevel, setApiServiceLevel] = useState(
     isLoadingSubscriptionData ? subscriptionApiServiceLevel : ApiServiceLevelEnum.FREE
   );
-  const [billingInterval, setBillingInterval] = useState<'month' | 'year'>('month');
+  const [billingInterval, setBillingInterval] = useState<'month' | 'year'>(subscriptionBillingInterval || 'month');
```

Comment:
We need intial `billingInterval` from an active subscription in order to not show "monthly" sub for existing "annual" customers